package.json

```diff
@@ -28,6 +28,7 @@
     "test:visualapprove": "backstop approve --config=tests/backstop/config.js",
     "test:analyze": "npm run build:production -- --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
     "test:js": "wp-scripts test-unit-js --config=tests/js/jest.config.js",
+    "test:js:coverage": "wp-scripts test-unit-js --config=tests/js/jest.config.js --coverage=true",
```

Comment:
This may be a good idea, however it's rather unrelated, so maybe better to remove?

Reply:
:+1: